feature-libs/cart/import-export/core/services/import-to-cart.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable, OnDestroy } from '@angular/core';
+import { SavedCartService } from '@spartacus/cart/saved-cart/core';
```

Comment:
After the SavedCartFacade was introduced in PR https://github.com/SAP/spartacus/pull/11949, we should not reference directly SavedCartService.